src/internal/connector/onedrive/collections_test.go

```diff
@@ -1236,7 +1258,9 @@ func (suite *OneDriveCollectionsSuite) TestGet() {
 			prevFolderPaths: map[string]map[string]string{
 				driveID1: {"root": rootFolderPath1},
 			},
-			expectedCollections: map[string]map[data.CollectionState][]string{},
+			expectedCollections: map[string]map[data.CollectionState][]string{
+				expectedPath1(""): {data.NotMovedState: {}},
```

Comment:
I know it is this way in some of the existing code too, but there is actually `rootFolderPath1` which can be used for root. There are a few instances of this.

```suggestion
				rootFolderPath1: {data.NotMovedState: {}},
```